pkg/gameserverallocations/metrics.go

```diff
@@ -54,12 +54,12 @@ func init() {
 
 // default set of tags for latency metric
 var latencyTags = []tag.Mutator{
-	tag.Insert(keyMultiCluster, "none"),
-	tag.Insert(keyClusterName, "none"),
-	tag.Insert(keySchedulingStrategy, "none"),
-	tag.Insert(keyFleetName, "none"),
-	tag.Insert(keyNodeName, "none"),
-	tag.Insert(keyStatus, "none"),
+	tag.Insert(keyMultiCluster, ""),
```

Comment:
I'm guessing now this is allowed to be empty ?

Reply:
Insert() should add this tag https://github.com/census-instrumentation/opencensus-go/blob/master/tag/map.go#L107:6
I will check tomorrow on how this applies to this metrics. Probably add a test for this.

Reply:
@cyriltovena This change break Stackdriver exporter, so I readded `"none"` for now. I think this change could be done separately.